sql/src/main/java/io/crate/metadata/rule/ingest/IngestRulesMetaData.java

```diff
@@ -117,18 +117,31 @@ public void createIngestRule(String sourceIdent, IngestRule ingestRule) throws I
         ingestRules.add(ingestRule);
     }
 
-    public void dropIngestRule(String ruleName) {
+    public void dropIngestRules(Set<String> rulesToDrop) {
         for (Set<IngestRule> ingestRules : sourceIngestRules.values()) {
             Iterator<IngestRule> iterator = ingestRules.iterator();
             while (iterator.hasNext()) {
                 IngestRule rule = iterator.next();
-                if (rule.getName().equals(ruleName)) {
+                if (rulesToDrop.contains(rule.getName())) {
                     iterator.remove();
-                    return;
+                    rulesToDrop.remove(rule.getName());
                 }
             }
         }
-        throw new IllegalArgumentException("Ingest rule " + ruleName + " doesn't exist");
+
+        if (rulesToDrop.isEmpty() == false) {
+            Iterator<String> iterator = rulesToDrop.iterator();
+            if (rulesToDrop.size() == 1) {
+                throw new IllegalArgumentException("Ingest rule " + iterator.next() + " doesn't exist");
+            } else {
+                StringBuilder missingRulesBuilder = new StringBuilder(iterator.next());
+                while (iterator.hasNext()) {
+                    missingRulesBuilder.append(", ");
+                    missingRulesBuilder.append(iterator.next());
+                }
+                throw new IllegalArgumentException("Ingest rules " + missingRulesBuilder.toString() + " don't exist");
+            }
+        }
```

Comment:
I'd prefer implementing a new separate method to drop rules for a table_ident and leave this method as it is